api/apps/api/src/modules/legacy-project-import/application/mark-legacy-project-import-as-finished.handler.ts

```diff
@@ -61,5 +67,11 @@ export class MarkLegacyProjectImportAsFinishedHandler
         ownerId,
       },
     });
+
+    await this.usersRepo.save({
+      userId: ownerId,
+      projectId: projectId.value,
+      roleName: ProjectRoles.project_owner,
+    });
```

Comment:
no need to change anything, especially at this stage - just noting, this would be a great candidate for a small expressive function to remind our future selves of what is the purpose of these lines 😱 - `setInitialOwnerForImportedProject()` or something like this. Let's leave it as is in any case - we do have a few other similar occurrences around, and it would only make sense to touch them all at once - but definitely not at this stage 😱 😅